packages/pasteup/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# What goes in pasteup?
+Anything that either generates style primitives or can be used to generate style primitives and is plain javascript (including JSON). For the most part this is pretty unambiguous: Colours, sizes, grid, etc all fit the bill. Font mappings are a good candidate too. Do NOT use paste up for components (as of Feb ‘19).
+Think of pasteup as a base foundation you can use to build components with the tech stack of your choice.
+
+# Releasing pasteup
+After your PR gets approved, but before merging run
+
+```
+make release-pasteup
+```
+This will ask you to up the version of pasteup. We use (semver)[https://semver.org] and you can check a quick reference below.
+
+That's it! Pasteup is now in [NPM](https://www.npmjs.com/package/@guardian/pasteup). There's a couple loose ends we need to deal with to tidy everything up:
+
+Your branch witll have a commit with a change to pasteup's `package.json` reflecting the new version. Right now we use alpha versions so you have to update the dependency version in `package.json` in `guui` and `frontend` too. With that done, push your changes to the packages & merge.
+
+Your changes will ripple to dotcom-rendering immediately. For other projects you must notify other consumers of the library of the updated version. At the moment that includes:
+
+- support-frontend (reader revenue teams)
+
+TBD: Should we create an email address? Do we have one?
+TBD: Should we release via riffraff or travis? What are the downsides? How hard is this to set up?
```

Comment:
https://github.com/guardian/mobile-apps-article-templates publish the templates package to NPM as part of CI. They're building on teamcity and pushing to npm.

Reply:
I can help set this up.

Reply:
omg that would be pretty awesome??